### PR TITLE
DEV-11140: Show job name in "toast" (message popup) when creating / editing

### PR DESCRIPTION
**Pull request by kevin-alpine**
This is a smallish change: 
* a few keys in the Messages_en.properties file
* A tweak to a backbone file ...
See original ticket here: https://alpine.atlassian.net/browse/DEV-11140

### CODE REVIEW

File: public/messages/Messages_en.properties [hunk 1]
```diff
@@ -1321,8 +1321,8 @@ job.dialog.meridiem.pm=pm
 job.dialog.next_run_date=Next run date / time:
 job.dialog.run_every=Run Every
 job.dialog.end_date_enabled=Disable this job on
-job.dialog.create.toast=Job created
-job.dialog.edit.toast=Job updated
+job.dialog.create.toast=Job {{jobName}} created
+job.dialog.edit.toast=Job {{jobName}} updated
```
Comment (kevin-alpine) [inline public/messages/Messages_en.properties]: Hi Atul,

Ok -- for this branch, this is kind of a nitpick, but I loaded this up in my dev build, and I noticed that, for these new messages, when you create and edit a job, the name of the job is displayed at the normal weight:

alpine chorus

... whereas, when you delete the job, the name of the job is bolded:

alpine chorus-1

The message for deleting a job has some HTML inline: `job_delete.toast=Job <span class="message_callout">{{jobName}}</span> has been deleted.`

Another minor comment, I'm not sure if there's some system to the naming of these keys.  Why `job.dialog.create.toast` vs `job_delete.toast`?  They are probably somewhat arbitrary, but might be worth asking @jleealpine if there's a philosophy behind this.

Comment (mmilano) [inline public/messages/Messages_en.properties]: @atul-alpine 
@kevin-alpine thanks, good catch. the correct pattern is to include the wrapper class for the item name. currently, that class will display in bold. (turns out that the toast library we use allows that sort of thing, and not other sorts of things. oh well).

re: key names. 
the key names have been growing organically over some years. no real logic or intelligent creator plan behind them, although one is wished for.

the inclusion of ".dialog" is inconsistent in the list. 

it is hoped to include some indication of the status for the toast: success, or neutral for neutral message types. see others for examples of this. (eg. insight.unpublish.success.toast)

**Comment by atul-alpine** [comment]
@kevin-alpine @mmilano please checkout latest commit for the changes in key-names and addition of wrapper class.

**Comment by kevin-alpine** [comment]
Perfect -- thanks @mmilano for the input, and thanks @atul-alpine for making the look of those messages consistent.

This PR now LGTM!

**Comment by kevin-alpine** [comment]
@prakash-alpine -- this PR is ready to be merged. Do you approve merging it in?

**Comment by prakash-alpine** [comment]
LGTM

**Pull request merged**